firestore-leaderboard/extension/functions/__tests__/functions.test.ts

```diff
@@ -0,0 +1,97 @@
+import mockedEnv from "mocked-env";
```

Comment:
I think this file needs a copyright and license header, too.

Reply:
done